lib/Differentiator/ActivityAnalyzer.h

```diff
@@ -0,0 +1,77 @@
+#ifndef CLAD_DIFFERENTIATOR_ACTIVITYANALYZER_H
+#define CLAD_DIFFERENTIATOR_ACTIVITYANALYZER_H
+
+#include "clang/AST/RecursiveASTVisitor.h"
+#include "clang/Analysis/CFG.h"
+
+#include "clad/Differentiator/CladUtils.h"
+#include "clad/Differentiator/Compatibility.h"
+
+#include <algorithm>
+#include <iterator>
+#include <memory>
+#include <set>
+#include <utility>
+/// @brief Class that implemets Varied part of the Activity analysis.
```

Comment:
```suggestion
/// Class that implemets Varied part of the Activity analysis.
```

No need of brief - it is deprecated and default now.